buildSrc/src/test/resources/rest/transform/feature/with_multiple_feature.yml

```diff
@@ -0,0 +1,13 @@
+---
+setup:
+  - skip:
+      features:
+      - pre_existing_feature1
+      - pre_existing_feature2
+---
+"Test without with multiple feature setup":
```

Comment:
is the `without` redundant?

Reply:
yup, thanks ! lots of copy paste happening here: fixed on [`bc7b2aa` (#68944)](https://github.com/elastic/elasticsearch/pull/68944/commits/bc7b2aa46fef51085bb1fd0293cb05be5a4b5ba4)